tests/presenters/teaser-presenter.spec.js

```diff
@@ -12,6 +12,21 @@ describe('Teaser Presenter', () => {
 
 	context('classModifiers', () => {
 
+		it('produces an empty array if mods undefined', () => {
+			const content = { mods: undefined };
+			subject = new Presenter(content);
+			expect(subject.classModifiers).to.deep.equal([]);
+		});
+
+		it('does not save previous teaser modifiers to new teaser', () => {
+			const content = { isOpinion: true };
+			let teaser1 = new Presenter(content);
+			let teaser2 =new Presenter(content);
```

Comment:
I disagree - I think that the reference to the mods is retained even when the Object.assign happens and is also assigned to `this.data.mods` in the presenter.

And in your instance you were repeatedly pushing the same mods array into each new presenter which was being modified with the classModifiers.